packages/myst-cli/src/store/reducers.ts

```diff
@@ -39,18 +39,13 @@ export const config = createSlice({
   } as {
     currentProjectPath: string | undefined;
     currentSitePath: string | undefined;
-    rawConfigs: Record<string, Record<string, any>>;
+    rawConfigs: Record<string, { raw: Record<string, any>; validated: Record<string, any> }>;
```

Comment:
Previously, the "raw config" that we saved in the store was a partially validated/coerced version of the actual config on disk. That meant to see if the config on disk changed, we had to re-validate before comparing.

Now, we still save the validated config (for using everywhere that we want a validated config), but we also save the totally raw, unvalidated config which we can compare to the file and check for changes without re-running any validation code.

Reply:
Nice, this will be helpful in the future as well!